bower.json

```
@@ -12,5 +12,45 @@
     "adapt-plugin",
     "adapt-component"
   ],
-  "license": "GPL-3.0"
+  "license": "GPL-3.0",
```

```suggestion
  "private": true,
  "license": "GPL-3.0",
```
Otherwise it'll get released on npm.